### PR TITLE
Update sitl-native-on-windows.rst

### PR DESCRIPTION
**Pull request by kd0aij**
Add pointers to instructions for running SITL using WSL

### CODE REVIEW

File: dev/source/docs/sitl-native-on-windows.rst [hunk 1]
```diff
@@ -4,9 +4,13 @@
 Setting up SITL on Windows
 ==========================
 
-SITL was originally developed on Linux, but can also be built and run *natively* Windows. It can also be run on a virtual machine (Linux) hosted on Windows, Mac OSX, or Linux.
+SITL was originally developed on Linux, but can also be built and run *natively* on Windows. It can also be run on a virtual machine (Linux) hosted on Windows, Mac OSX, or Linux or by using the Windows Subsytem for Linux (WSL).
 
-This article shows how to build and run :ref:`SITL (Software In The Loop) <sitl-simulator-software-in-the-loop>` *natively* on Windows using Cygwin. These commands were tested on Windows 10 with :ref:`MAVProxy <mavproxy:home>` and Copter-4.0.0-dev.
+Currently, the simplest way to run SITL on Windows is to use WSL: https://ardupilot.org/dev/docs/building-setup-windows10.html?highlight=wsl#using-sitl-with-wsl-and-xwindows
```
Comment (Hwurzburg) [inline dev/source/docs/sitl-native-on-windows.rst:9]:
```suggestion
Currently, the simplest way to run SITL on Windows is to use build the code and simulate using WSL:  :ref:`building-setup-windows10`
```

File: dev/source/docs/sitl-native-on-windows.rst [hunk 2]
```diff
@@ -4,9 +4,13 @@
...
+
+To set up WSL follow these instructions: https://ardupilot.org/dev/docs/building-setup-windows10.html?highlight=wsl#setting-up-the-build-environment-on-windows10-using-wsl1-or-wsl2
```
Comment (Hwurzburg) [inline dev/source/docs/sitl-native-on-windows.rst:11]: remove line 11...dupication

File: dev/source/docs/sitl-native-on-windows.rst [hunk 3]
```diff
@@ -4,9 +4,11 @@
 Setting up SITL on Windows
 ==========================
 
-SITL was originally developed on Linux, but can also be built and run *natively* Windows. It can also be run on a virtual machine (Linux) hosted on Windows, Mac OSX, or Linux.
+SITL was originally developed on Linux, but can also be built and run *natively* on Windows. It can also be run on a virtual machine (Linux) hosted on Windows, Mac OSX, or Linux or by using the Windows Subsytem for Linux (WSL).
 
-This article shows how to build and run :ref:`SITL (Software In The Loop) <sitl-simulator-software-in-the-loop>` *natively* on Windows using Cygwin. These commands were tested on Windows 10 with :ref:`MAVProxy <mavproxy:home>` and Copter-4.0.0-dev.
+Currently, the simplest way to run SITL on Windows is to build the code and simulate using WSL:  :ref:`dev:building-setup-windows10`
+
```
Comment (hendjoshsr71) [inline dev/source/docs/sitl-native-on-windows.rst:10]: I would add a `::note ` around these parts with a warning about only using cygwin for building mission planner binaries.

**Comment by hendjoshsr71** [comment]
Not for this PR:
I'd be tempted to suggest deeper cuts here. A lot of the info on these Using SITL pages is better classified as how to setup the build environment.

I'm not sure how the "Using SITL" part is different enough between OSs to warrant separate pages. It seems like a lot of folks come right to these pages and skip the Installing the "Build Environment" pages.

I'd have one Using SITL page the with a single section on where to go for building the right environment.

File: dev/source/docs/sitl-native-on-windows.rst (hunk 3 above)
Comment (Hwurzburg) [reply dev/source/docs/sitl-native-on-windows.rst:10]: going to merge and add note immediately after....

**Pull request merged**